lib/custom_widgets/primary_products.dart

```diff
@@ -4,24 +4,24 @@ import 'package:flutter/material.dart';
 class PrimaryProduct extends StatefulWidget {
   final Color color;
   final String title;
-  final String image;
-  final String image1;
-  final String image2;
-  final String image3;
-  final String image4;
+  final String flowerImage;
+  final String shapeImage;
+  final String footImage;
+  final String iconImage;
+  final String heartImage;
   final String subtitle;
-  final String subtitle2;
+  final String secondsubtitle;
```

Comment:
Instead of Second use secondary but secondary is still not good, may be label is a better option